analytics/location_heatmaps/geo_utils.py

```diff
@@ -278,141 +296,124 @@ def split_regions(tree_prefix_list,
   Args:
       tree_prefix_list: matches vector id to the tree prefix.
       vector_counts: vector values aggregated from the users.
-      threshold: threshold value used to split the nodes.
+      split_threshold: threshold value used to split the nodes.
       image_bit_level: stopping criteria once the final resolution is reached.
       collapse_threshold: threshold value used to collapse the nodes.
+      expand_all: expand all regions,
+      last_result: use previous level results to compute conf intervals,
+      count_min: use count-min sketch
   Returns:
-      new_tree, new_tree_prefix_list, finished
+      new_tree, new_tree_prefix_list, fresh_expand
   """
   collapsed = 0
   created = 0
   fresh_expand = 0
   unchanged = 0
```

Comment:
collapsed, created, and unchanged do not appear to be used for anything anymore. let's delete them xor do something with them.

Reply:
done

Reply:
printing the results in the end of the function now